finality-aleph/src/finalization.rs

```diff
@@ -195,7 +192,7 @@ impl<
     pub(crate) fn new(network: RmcNetwork<B>, keychain: &'a MK) -> Self {
         let (messages_for_rmc, messages_from_network) = mpsc::unbounded();
         let (messages_for_network, messages_from_rmc) = mpsc::unbounded();
-        let scheduler = DoublingDelayScheduler::new(Duration::from_millis(10));
+        let scheduler = DoublingDelayScheduler::new(Duration::from_millis(500));
```

Comment:
Maybe this could be some parameter of finalization or BlockSignatureAggregator?

Reply:
This is a very low-level parameter, which we most likely don't want to play with. So I would leave it as it is...